uqcsbot/scripts/holiday.py

```diff
@@ -76,6 +79,21 @@ def get_holidays_from_page(holiday_page) -> list:
     return holidays
 
 
+def get_holidays_from_csv():
```

Comment:
Can we get some typing on this function?

`def get_holidays_from_csv() -> List[Something]`

You may have to import complex types from the `typing` module.

Reply:
![image](https://user-images.githubusercontent.com/56781181/83591867-4fc4cc00-a59c-11ea-9b4f-76491d1b7b1c.png)
I'm not certain what you wanted a screenshot of. Here is the terminal output, you can see it runs successfully. The warning at the end is because I exited with a keyboard interrupt.
![image](https://user-images.githubusercontent.com/56781181/83591960-8a2e6900-a59c-11ea-9415-76725fcdbc04.png)
This image is from the testing slack. There are two because I ran it twice, once to check that the original website scraping holidays work and the other to test a holiday from the `geek_holidays.csv`.

Reply:
Also, see the commit below for the type hinting I added to `holiday.py`.

Reply:
The screenshot of it working is usually all we're after.